### ISSUE TITLE
Tabs activating as you progress specifying the model

### ISSUE THREAD

**Issue by nuferates** [aalfons/robmedExtra#23]
First I acknowledge that the GUI looks great and I did not experience any problems using it. It was straightforward and easy to use. So the issues I enter here are mostly 'minor points' and just 'some food for thought'. 

The tab structure is very intuitive. It flows from left to right. The user starts with the data tab and moves to the next tab on the right. 

I also acknowledge that there are already red warnings such as "Select a dependent variable, at least one independent variable, and at least one mediator in the Model tab" and "Run ROBMED or the OLS bootstrap in the respective tabs" in the Analyses tabs and the Export tab respectively -which is very very informative. 

In line with this, just as a minor point, I suggest the tabs on the right can be 'gray' and/or 'unclickable' before the information on the left tab is entered (Of course except for the Info tab). Once Data is chosen (or uploaded), the Model tab activates. Once a Model is specified, the ROBMED and OLSBootstrap tabs get activated. Once the results are produced in ROBMED or OLSBootstrap tabs, the Export tab gets activated.

**Comment by aalfons** [comment]
One issue may be that it may not be clear to the user why some tabs are not clickable. So we would also need some help text or something that is shown, for example when the cursor hovers over an inactive tab.

With the current solution, if a user clicks on a tab, they get a message of what they need to do first. So the order of tabs becomes very clear to the user when they use the interface for the first time. I'm therefore not sure if deactivating the tabs would add much. 

So this is not a priority, and we should only change this if it is very easy to do. @AuroreAA, do you know if it is easy from a technical perspective to deactivate certain tabs and activate them when a certain condition is met?

**Comment by AuroreAA** [comment]
Yes it is possible (quite easily). But I agree with you @aalfons we have warning messages each time so I think it is fine like that. 
However, another solution I proposed in another issue could be convenient. On each tab we can add a button "to the next step" which opens the next tab...
